mantidimaging/core/operations/roi_normalisation/roi_normalisation.py

```diff
@@ -46,8 +47,16 @@ def filter_func(images: Images, air_region: SensibleROI = None, cores=None, chun
 
         # just get data reference
         if air_region:
+            initial_image_max = images.data.max()
+
             progress = Progress.ensure_instance(progress, task_name='ROI Normalisation')
             _execute(images.data, air_region, cores, chunksize, progress)
+
```

Comment:
```suggestion
            progress.update(1, "Rescaling to input value range")
```
Updates the progress message to say that the data is being rescaled, otherwise it stays on "complete" for a few seconds